lmdeploy/pytorch_poc/engine/engine.py

```diff
@@ -456,14 +462,14 @@ class Engine:
         tp (int): Number of tensor parallel.
     """
 
-    def __init__(
-        self,
-        model_path: str,
-        scheduler_config: SchedulerConfig = None,
-        cache_config: CacheConfig = None,
-        tp: int = 1,
-        trust_remote_code=True,
-    ) -> None:
+    def __init__(self,
+                 model_path: str,
+                 scheduler_config: SchedulerConfig = None,
+                 cache_config: CacheConfig = None,
+                 tp: int = 1,
+                 trust_remote_code=True,
+                 json_config_file: str = 'config.json',
+                 max_session_len=2048) -> None:
```

Comment:
max_session_len can be passed by scheduler_config

Reply:
fixed.